src/components/GitmojiList/Toolbar/ListModeSelector/styles.module.css

```
@@ -9,7 +9,7 @@
   color: var(--text);
   cursor: pointer;
   display: flex;
-  background-color: transparent;
+  background-color: var(--buttonBackground);
```

I think we can use this value here to keep everything consistent between the cards for the emojis and for the buttons 

```suggestion
  background-color: var(--cardBackground);
```